cmd/jujud/bootstrap_test.go

```diff
@@ -866,7 +866,7 @@ func (s *BootstrapSuite) makeTestEnv(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 
 	s.PatchValue(&juju.JujuPublicKey, sstesting.SignedMetadataPublicKey)
-	envtesting.MustUploadFakeTools(s.toolsStorage, cfg.AgentStream(), cfg.AgentStream())
+	envtesting.MustUploadFakeToolsToSimpleStreams(s.toolsStorage, cfg.AgentStream(), cfg.AgentStream())
```

Comment:
can we also then call s.toolStorage "s.simplestreamsToolsStorage" to disambiguate from tools storage used elsewhere